### PR TITLE
Various fixes

### PR DESCRIPTION
**Pull request by nnhw**
A collection of changes, made into the jerom branch

### CODE REVIEW

File: scripts/ubuntu_install_deps.sh [hunk 1]
```diff
@@ -8,5 +8,5 @@ else
 fi
 
 $SUDO apt update -qq
-$SUDO apt install -y nodejs npm cmake google-mock googletest qttools5-dev qttools5-dev-tools qtbase5-dev qtchooser qtdeclarative5-dev qtmultimedia5-dev qtquickcontrols2-5-dev libqt5serialport5-dev libqt5websockets5-dev libqt5svg5-dev libqt5opengl5-dev libqt5sql5-psql qml-module-qtquick-controls2 qml-module-qtmultimedia qml-module-qtgraphicaleffects qml-module-qtquick-layouts qml-module-qt-labs-settings qml-module-qtquick-dialogs qml-module-qtwebchannel qtwebengine5-dev qml-module-qt-labs-qmlmodels qml-module-qtwebengine
```
Comment (MishkaRogachev) [inline scripts/ubuntu_install_deps.sh:11]: Are we still need google-mock googletest libgtest-dev in script?

File: .gitignore [hunk 2]
```diff
@@ -57,6 +57,12 @@ compile_commands.json
 /node_modules/
 /Debug
 /Release
+/modules/web
```
Comment (MishkaRogachev) [inline .gitignore:60]: There are cases when garbage appears in /modules/web and /modules/Cesium

Comment (nnhw) [reply .gitignore:60]: Does it mean that we should add this lines?

File: scripts/ubuntu_install_deps.sh (hunk 1 above)
Comment (nnhw) [reply scripts/ubuntu_install_deps.sh:11]: No, I'll clean it up

File: .gitignore (hunk 2 above)
Comment (nnhw) [reply .gitignore]: Removed, due to possible existance of the srs in this directories

**Pull request merged**